big_tests/tests/mod_ping_SUITE.erl

```diff
@@ -134,7 +157,10 @@ server_ping_pong(Config) ->
 
 server_ping_pang(ConfigIn) ->
     Domain = ct:get_config({hosts, mim, domain}),
-    Metrics = [{[Domain, user_ping_timeout], 1}],
+    Metrics = [
+        {[Domain, mod_ping, ping_response], 0},
+        {[Domain, mod_ping, ping_response_timeout], 1}],
+%%      {[Domain, mod_ping, ping_response_time, 95],  {1, 1000000000}}], % not possible to compare histograms in escalus
```

Comment:
Not entirely true. I think it's not possible to compare specific histogram properties but it's definitely possible to check if there were any histogram updates, e.g. https://github.com/esl/MongooseIM/blob/master/big_tests/tests/metrics_roster_SUITE.erl#L67